plugins/notifications-backend/src/service/router-test-ts

```diff
@@ -0,0 +1,53 @@
+import { getVoidLogger } from '@backstage/backend-common';
```

Comment:
Filename is a bit wrong

Reply:
this file was left by mistake. will be removed.